src/Controller/OcrController.php

```diff
@@ -177,8 +178,40 @@ public function homeAction(): Response
     }
 
     /**
-     * @Route("/api", name="api")
-     * @Route("/api.php", name="apiPhp")
+     * @Route("/api", name="api", methods={"GET", "POST"})
+     * @Route("/api.php", name="apiPhp", methods={"GET", "POST"})
+     * @OA\Parameter(
+     *     name="engine",
+     *     in="query",
+     *     description="The engine to use, either `tesseract` or `google`.",
```

Comment:
The interactive input field for this is using "$name - $description" as placeholder, which doesn't fit into the field. Would it be possible to set it to some other value, e.g. "tesseract"? Same for the other two fields.